### PR TITLE
refactor: Remove require and @require syntax support

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: src/main/java/com/virtuslab/using_directives/custom/Parser.java [hunk 1]
```diff
@@ -112,13 +112,9 @@ UsingDefs usingDirectives() {
...
       int offset = offset(in.td.offset);
-
-      UsingDirectiveSyntax syntax = UsingDirectiveSyntax.Using;
-      if (in.td.token == Tokens.ATREQUIRE) syntax = UsingDirectiveSyntax.AtRequire;
-      else if (in.td.token == Tokens.REQUIRE) syntax = UsingDirectiveSyntax.Require;
-
       in.nextToken();
-      return new UsingDef(settings(), syntax, source.getPositionFromOffset(offset));
+      return new UsingDef(
+          settings(), UsingDirectiveSyntax.Using, source.getPositionFromOffset(offset));
```
Comment (szymon-rd) [inline src/main/java/com/virtuslab/using_directives/custom/Parser.java]: I think there is no point in keeping the `UsingDirectiveSyntax` as enum and passing it as there is only one possible value now.

File: src/main/java/com/virtuslab/using_directives/custom/model/UsingDirectiveSyntax.java [hunk 2]
```diff
@@ -1,9 +1,7 @@
 package com.virtuslab.using_directives.custom.model;
 
 public enum UsingDirectiveSyntax {
```
Comment (szymon-rd) [inline src/main/java/com/virtuslab/using_directives/custom/model/UsingDirectiveSyntax.java]: I would get rid of this enum

File: src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java [hunk 3]
```diff
@@ -32,9 +32,7 @@ public static boolean isIdentifier(Tokens token) {
   }
 
   public static boolean isValidUsingDirectiveStart(Tokens token, Settings settings) {
-    return (settings.isAllowStartWithoutAt()
-            && (token == Tokens.USING || (settings.isAllowRequire() && token == Tokens.REQUIRE)))
-        || (settings.isAllowRequire() && token == Tokens.ATREQUIRE);
+    return (settings.isAllowStartWithoutAt() && token == Tokens.USING);
```
Comment (szymon-rd) [inline src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java]: It seems that `isAllowStartWithoutAt` is redundant now as well. It's the only allowed way if I understand correctly.

**Review by szymon-rd — CHANGES_REQUESTED** [review]

File: src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java (hunk 3 above)
Comment (tgodzik) [reply src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java]: Removed

File: src/main/java/com/virtuslab/using_directives/custom/model/UsingDirectiveSyntax.java (hunk 2 above)
Comment (tgodzik) [reply src/main/java/com/virtuslab/using_directives/custom/model/UsingDirectiveSyntax.java]: Removed and also Settings, which is no longer needed

File: src/main/java/com/virtuslab/using_directives/custom/Parser.java (hunk 1 above)
Comment (tgodzik) [reply src/main/java/com/virtuslab/using_directives/custom/Parser.java]: Yep, removed.

**Review by szymon-rd — APPROVED** [review]

**Pull request merged**